GOVERNANCE.md

```diff
@@ -94,11 +64,74 @@ Other, less formal, kinds of contribution are outlined in our
+- **Gold, Silver, and Bronze Sponsors**: these sponsorship levels correspond to different amounts of recurring financial contribution that generously support our contributor compensation scheme. Sponsors at these levels also have the opportunity to display their logo on our website.
+- **Backers**: individuals who benefit from the Appium project can also donate on a one-time or recurring basis, at any amount, and will be recognized as Backers for their contribution!
+
+To become a sponsor at one of these levels, follow the process indicated at the [OpenCollective
+page](https://opencollective.com/appium).
+
+### Compensation for TC Members, Committers, Contributors, and upstream projects
+
+With funds received through sponsorship, the Appium project wants to incentivize contributions of
+code, documentation efforts, maintenance, and project leadership. We have developed a scheme
+(inspired by one developed at the [WebdriverIO](https://github.com/webdriverio/webdriverio)
+project) for disbursing funds on a monthly basis to those who contribute to the project, as well as
+some other "upstream" open source projects.
+
+Each month, 15% of sponsorship funds received will be donated to upstream projects, based on the
+discretion of the current TC. At this point, no attempt will be made to formalize quantitative
+notions of relevance. If you manage a project upstream of Appium and would like to be considered as
+part of this donation group, give us a shout!
+
+Of the remaining 85% of sponsorship funds, 70% (roughly 60% of the original total) will go to
+project Committers (i.e., those in a role of maintenance), and 30% (roughly 25% of the original
+total) will go to Contributors (i.e., those making one-off or periodic contributions). The way that
+Committers and Contributors are paid for their efforts differs.
+
+For Committers, each Committer will keep a monthly time sheet of hours contributed to the project.
+At the end of each month, the hours tracked from all Committers will be totalled, and then the
+Committer-allocated funds will be disbursed (via OpenCollective) on a pro rata basis. Within this
+scheme, Committers agree not to submit hours worked as part of their employment with a Development
+Partner (since this is part of the "sponsorship" contribution of the Development Partner).
+
+For Contributors, when they make contributions which are successfully merged into the project,
+a Committer can discretionarily assign a "value tier" to the contribution. The "value tiers" and
+corresponding payouts are as follows:
+
+- XS: a trivial change (e.g. a typo change or applying of an automatically suggested security
```

Comment:
Does github have a possibility to create a filter of matching PRs made within a month or a dashboard, so we could automate the estimation process?

Reply:
also not sure how open if the estimation process should be.
Perhaps, we could simply add appropriate tags to PRs to make that more explicit

Reply:
I agree, tags seem like a reasonable option here